modules/categories.js

```diff
@@ -0,0 +1,13 @@
+const categoryId = entered_category => {
```

Comment:
This is level, right? Can we change the filename to levels.js then name teh functions appropriately and ensure index.js is also updated properly.

Reply:
OK